spring-boot-samples/spring-boot-sample-cache-ehcache/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.cache.type=ehcache
```

Comment:
I would actually *not* put this. This proves that ehcache is detected as it should because it's in the classpath.